nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/actions/SetSystemPropertiesAction.java

```diff
@@ -45,15 +56,33 @@ protected Computation createComputation(int batchSize, int batchThresholdMs) {
 
     public static class SetSystemPropertyComputation extends AbstractBulkComputation {
 
+        public static final String NOTIFY = "param-notify";
+
         public SetSystemPropertyComputation(String name, int batchSize, int batchThresholdMs) {
             super(name, 1, 1, batchSize, batchThresholdMs);
         }
 
         @Override
         protected void compute(CoreSession session, List<String> ids, Map<String, Serializable> properties) {
-            ids.forEach(id -> properties.forEach((k, v) -> session.setDocumentSystemProp(new IdRef(id), k, v)));
+            Map<String, Serializable> actualProperties = new HashMap<>(properties);
+            String eventId = (String) actualProperties.remove(NOTIFY);
```

Comment:
Why all this code to get a flag which is not event an eventId ?

Reply:
It's because we have to remove and get NOTIFY to iterate on all other and the provided map is unmodifiable
We could also jut get and put a filter in the stream line 70
Which one do you prefer ?

Reply:
ok, I see, I don't think it is going to be a pb so no change required.